### ISSUE TITLE
[internal testing] Debug failed with error tips "Invalid 'cwd' value ' C:\Windows\System32\MSVCprojects\helloworld\helloworld.cpp'. The system cannot find the path specified.  " 

### ISSUE THREAD

**Issue by Wanye1062271859** [microsoft/vscode-cpptools#4624]
According to the tutorial: https://code.visualstudio.com/docs/cpp/config-msvc, Debugging the sample project after creating helloworld.cpp, the error " **Invalid 'cwd' value ' C:\Windows\System32\MSVCprojects\helloworld\helloworld.cpp'. The system cannot find the path specified.** " pops up, actually the helloworld.cpp is existing.

![image](https://user-images.githubusercontent.com/29274719/69030311-24b8c780-0a12-11ea-9d50-67bc1829509c.png)

VSCode version: 1.40.1
C/C++ extension version: 0.26.2
VS version: 16.4.0 Preview 5.0 [29512.175]

**Comment by pieandcakes** [comment]
@Wanye1062271859 Hi,

`cwd` stands for current working directory and should not point to a file. This is the correct error. If you expect a different error, please let me know.

**Comment by Wanye1062271859** [comment]
Hi pieandcakes,
Thanks for your reply. This issue has been resolved. But the cwd path does not point to a file is not the cause of the error.  I tried change the value of the cwd to "C:\\Windows\System32\projects\helloworld\helloworld.cpp", this issue still reproduce. 

The real reason is because my workspace was set to C:\Windows\System32\...., There seem to be some permissions access issues here. I changed the workspace to another path, this issues is no longer reproduce.
